app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split()
```

Comment:
It is not a good naming, because you will have also a command name in the "files", that's not obvious